tensorflow/python/kernel_tests/nn_ops/pooling_ops_3d_test.py

```diff
@@ -104,6 +112,9 @@ def _VerifyValues(self, pool_func, input_sizes, window, strides,
       if use_gpu and test_util.is_gpu_available(cuda_only=True):
         self._VerifyOneTest(pool_func, input_sizes, window, strides, padding,
                             data_format, dtypes.bfloat16, expected, use_gpu)
+      elif not use_gpu:
+        self._VerifyOneTest(pool_func, input_sizes, window, strides, padding,
+                            data_format, dtypes.bfloat16, expected, use_gpu)
```

Comment:
The only case where bf16 isn't run is when `use_gpu` is `true` and `is_gpu_available` is `false`. I think we can just merge the two `if`s.
```suggestion
      # Don't test bfloat16 on GPU if there is no GPU available.
      if (not use_gpu) or test_util.is_gpu_available(cuda_only=True):
        self._VerifyOneTest(pool_func, input_sizes, window, strides, padding,
                            data_format, dtypes.bfloat16, expected, use_gpu)
```